src/dipdup/cli.py

```diff
@@ -111,11 +111,15 @@ def _get_paths(
 
 
 def _load_env_files(env_file_paths: list[Path]) -> None:
-    for path in env_file_paths:
-        from dotenv import load_dotenv
```

Comment:
Import inside the loop is intentional. This way dotenv library won't be imported if not used.